data/newMigrations/010-recipients.js

```diff
@@ -0,0 +1,39 @@
+exports.up = (knex) => {
+    return knex.schema.createTable('recipients', function (tbl) {
+        tbl.uuid('recipient_id').primary();
+        tbl.string('recipient_first_name');
+        tbl.string('recipient_middle_name');
+        tbl.string('recipient_last_name');
+        tbl.date('recipient_date_of_birth');
+        tbl.boolean('recipient_veteran_status');
+        tbl.uuid('household_id')
+            .unsigned()
+            .references('household_id')
+            .inTable('households')
+            .onUpdate('RESTRICT')
+            .onDelete('RESTRICT');
+        tbl.int('gender_id')
```

Comment:
Mistake on the schema on my part! Everywhere where it says int in the schema, you should actually be using `.integer` not `.int`. I don't think `.int` is a valid knex command.;